e2e/tests/plugins/imagery/exampleImagery.e2e.spec.js

```diff
@@ -200,23 +200,154 @@ test.describe('Example Imagery', () => {
+// ('Can zoom into the latest image and the real-time/fixed-time imagery will pause');
+// ('Clicking on the left arrow should pause the imagery and go to previous image');
+// ('If the imagery view is in pause mode, it should not be updated when new images come in');
+// ('If the imagery view is not in pause mode, it should be updated when new images come in');
+const backgroundImageSelector = '.c-imagery__main-image__background-image';
+test('Example Imagery in Display layout', async ({ page }) => {
+
+    page.on('console', msg => console.log(msg.text()));
+    // Go to baseURL
+    await page.goto('/', { waitUntil: 'networkidle' });
+
+    // Click the Create button
+    await page.click('button:has-text("Create")');
+
+    // Click text=Example Imagery
+    await page.click('text=Example Imagery');
+
+    // Clear and set Image load delay (milliseconds)
+    await page.click('input[type="number"]', {clickCount: 3})
+    await page.type('input[type="number"]', "20")
+
+    // Click text=OK
+    await Promise.all([
+        page.waitForNavigation({waitUntil: 'networkidle'}),
+        page.click('text=OK'),
+        //Wait for Save Banner to appear
+        page.waitForSelector('.c-message-banner__message')
+    ]);
+    // Wait until Save Banner is gone
+    await page.waitForSelector('.c-message-banner__message', { state: 'detached'});
+    await expect(page.locator('.l-browse-bar__object-name')).toContainText('Unnamed Example Imagery');
+
+    const bgImageLocator = await page.locator(backgroundImageSelector);
+    const deltaYStep = 100; // equivalent to 1x zoom
+    const originalImageDimensions = await page.locator(backgroundImageSelector).boundingBox();
+    const selectedImage = page.locator('.selected');
+    const previousImageButton = await page.locator('.c-nav--prev');
+    const nextImageButton = await page.locator('.c-nav--next');
+    const panHotkey = process.platform === 'linux' ? ['Control', 'Alt'] : ['Alt'];
+    await bgImageLocator.hover();
+    
+    // Click previous image button
+    await previousImageButton.click();
+
+    // Verify previous image 
+    await expect(selectedImage).toBeVisible();
+
+    // Zoom in
+    await bgImageLocator.hover();
+    await page.mouse.wheel(0, deltaYStep * 2);
+    const zoomedBoundingBox = await bgImageLocator.boundingBox();
+    const imageCenterX = zoomedBoundingBox.x + zoomedBoundingBox.width / 2;
+    const imageCenterY = zoomedBoundingBox.y + zoomedBoundingBox.height / 2;
+
+    // Wait for zoom animation to finish
+    await bgImageLocator.hover();
+    const imageMouseZoomedIn = await page.locator(backgroundImageSelector).boundingBox();
+    expect(imageMouseZoomedIn.height).toBeGreaterThan(originalImageDimensions.height);
+    expect(imageMouseZoomedIn.width).toBeGreaterThan(originalImageDimensions.width);
+
+    // Center the mouse pointer
+    await page.mouse.move(imageCenterX, imageCenterY);
+
+    // Pan Imagery Hints
+    console.log(process.platform);
```

Comment:
I missed this in the intial code code review from the other test. Would you mind logging this like 
```js
console.log('process.platform is '+process.platform);
```

Reply:
Updated